diff -urN quodlibet-0.19.1.orig/const.py quodlibet-0.19.1/const.py
--- quodlibet-0.19.1.orig/const.py	2006-04-19 12:52:37.000000000 +0200
+++ quodlibet-0.19.1/const.py	2006-04-19 12:54:01.000000000 +0200
@@ -6,7 +6,7 @@
 VERSION = "0.19.1"
 
 HOME    = os.path.expanduser("~")
-USERDIR = os.path.join(HOME, ".quodlibet")
+USERDIR	= os.path.join(os.getenv("HOME_ETC",HOME),".quodlibet")
 CONTROL = os.path.join(USERDIR, "control")
 CONFIG  = os.path.join(USERDIR, "config")
 CURRENT = os.path.join(USERDIR, "current")
