Index: qltk/quodlibet.py
===================================================================
--- qltk/quodlibet.py	(revision 3998)
+++ qltk/quodlibet.py	(working copy)
@@ -223,9 +223,9 @@
         sw.set_shadow_type(gtk.SHADOW_IN)
         sw.add(self.songlist)
 
-        self.songpane = gtk.VBox(spacing=6)
-        self.songpane.pack_start(self.song_scroller)
-        self.songpane.pack_start(self.qexpander, expand=False, fill=True)
+        self.songpane = gtk.VPaned()
+        self.songpane.pack1(self.song_scroller, resize=True)
+        self.songpane.pack2(self.qexpander, resize=True)
         self.songpane.show_all()
         self.song_scroller.connect('notify::visible', self.__show_or)
         self.qexpander.connect('notify::visible', self.__show_or)
@@ -355,9 +355,6 @@
         ssv = self.song_scroller.get_property('visible')
         qxv = self.qexpander.get_property('visible')
         self.songpane.set_property('visible', ssv or qxv)
-        self.songpane.set_child_packing(
-            self.qexpander, expand=not ssv, fill=True, padding=0,
-            pack_type=gtk.PACK_START)
         if not ssv:
             self.qexpander.set_expanded(True)
 
