diff -Nur quodlibet-2.5.orig/quodlibet/util/modulescanner.py quodlibet-2.5/quodlibet/util/modulescanner.py
--- quodlibet-2.5.orig/quodlibet/util/modulescanner.py	2012-12-18 14:06:30.000000000 +0000
+++ quodlibet-2.5/quodlibet/util/modulescanner.py	2013-01-12 15:16:54.784695709 +0000
@@ -33,7 +33,7 @@
             continue
         if entry.endswith(".py"):
             modules.add(entry[:-3])
-        elif load_compiled and entry.endswith(".pyc"):
+        elif entry.endswith(".pyc"):
             modules.add(entry[:-4])
 
     loaded = []
