--- quodlibet-1.0/Makefile.wiget	2007-04-16 04:45:47.000000000 +0200
+++ quodlibet-1.0/Makefile	2007-05-07 18:32:26.000000000 +0200
@@ -10,15 +10,15 @@
 all:
 	@/bin/echo -n "Checking for Python... "
 	@which python || ( echo "Not found." && /bin/false )
-	@./check.py $(DESTDIR)$(PREFIX)/$(TO)
+	@./check.py $(DESTDIR)$(LIBDIR}/quodlibet
 
 make-install-dirs:
 	mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
-	mkdir -p $(DESTDIR)$(PREFIX)/$(TO)
+	mkdir -p $(DESTDIR)$(LIBDIR)/quodlibet
 
 install: make-install-dirs install-dirs install-programs
-	install -m 644 $(MODULES) missing-cover.svg audio-*.png media-*.png device-*.png $(DESTDIR)$(PREFIX)/$(TO)
+	install -m 644 $(MODULES) missing-cover.svg audio-*.png media-*.png device-*.png $(DESTDIR)$(LIBDIR)/quodlibet
 	cd po && make install-po DESTDIR=$(DESTDIR)
 
 	for E in $(EXTENSIONS); do \
@@ -32,16 +32,16 @@
 install-programs: $(addprefix app-install-, $(PROGRAMS))
 
 dir-install-%: make-install-dirs
-	mkdir -p $(DESTDIR)$(PREFIX)/$(TO)/$*
-	install -m 644 $*/*.py $(DESTDIR)$(PREFIX)/$(TO)/$*
+	mkdir -p $(DESTDIR)$(LIBDIR)/quodlibet/$*
+	install -m 644 $*/*.py $(DESTDIR)$(LIBDIR)/quodlibet/$*
 
 app-install-%: make-install-dirs %.desktop
-	install -m 755 $*.py $(DESTDIR)$(PREFIX)/$(TO)
+	install -m 755 $*.py $(DESTDIR)$(LIBDIR)/quodlibet
 	install -m 644 $*.1 $(DESTDIR)$(PREFIX)/share/man/man1/$*.1
 	install -D -m 644 $*.png $(DESTDIR)$(PREFIX)/share/pixmaps/$*.png
-	install -m 644 $*.svg $*.png $(DESTDIR)$(PREFIX)/$(TO)
+	install -m 644 $*.svg $*.png $(DESTDIR)$(LIBDIR)/quodlibet
 	-install -D -m 644 $*.desktop $(DESTDIR)$(PREFIX)/share/applications/$*.desktop
-	ln -sf ../$(TO)/$*.py $(DESTDIR)$(PREFIX)/bin/$*
+	ln -sf $(LIBDIR)/quodlibet/$*.py $(DESTDIR)$(PREFIX)/bin/$*
 
 clean:
 	rm -f *.py[co] */*.py[co] $(EXTENSIONS) messages.mo
