--- quodlibet-2.3/quodlibet/const.py.old	2011-04-01 00:44:00.000000000 +0200
+++ quodlibet-2.3/quodlibet/const.py	2011-04-03 18:38:47.397249347 +0200
@@ -24,7 +24,10 @@
         USERDIR = shell.SHGetFolderPath(0, shellcon.CSIDL_APPDATA, 0, 0)
         USERDIR = os.path.join(USERDIR, "Quod Libet")
     else:
-        USERDIR = os.path.join(HOME, ".quodlibet")
+        if 'HOME_ETC' in os.environ:
+            USERDIR = os.path.join(os.environ['HOME_ETC'], ".quodlibet")
+        else:
+            USERDIR = os.path.join(HOME, ".quodlibet")
 
 CONTROL = os.path.join(USERDIR, "control")
 CONFIG  = os.path.join(USERDIR, "config")
